src/Fields/fieldvector.jl

```diff
@@ -210,7 +194,7 @@ end
 transform_broadcasted(fv::FieldVector, symb, axes) =
     parent(getfield(_values(fv), symb))
 transform_broadcasted(x, symb, axes) = x
-Base.@propagate_inbounds function Base.copyto!(
+@inline function Base.copyto!(
```

Comment:
We wan to keep `@propagate_inbounds` it also inline annotates the method call

Reply:
I thought I did it somewhat consistently for most of the datalayouts.  You don't have to do it in this PR but we should clean it up at some point (it is annoying).

we put in `@inbounds` into toplevel `copyto!` calls